lib/rubocop/cop/style/multiline_memoization.rb

```diff
@@ -71,6 +86,15 @@ def bad_rhs?(rhs)
             rhs.kwbegin_type?
           end
         end
+
+        def newline_after?(node, pos)
+          node.source_range.source_buffer.source[pos] == "\n"
```

Comment:
Actually I think the code needs to be more complicated. :smile: @nikolai-b Please add comments after the opening `(` in the "code with offense" examples and make sure the comment ends up in the corresponding position in the auto-corrected code.